srcs/juloo.keyboard2/KeyValue.java

```diff
@@ -551,6 +551,8 @@ public static KeyValue getKeyByName(String name)
       /* intending/preventing ligature - supported by many scripts*/
       case "zwj": return charKey("zwj", '\u200D', 0); // zero-width joiner (provides ligature)
       case "zwnj": return charKey("zwnj", '\u200C', 0); // zero-width non joiner (prevents unintended ligature)
+      /* persian halfspace */
+      case "halfspace": return charKey("⸽", '\u200C', 0);
```

Comment:
The new `halfspace` key outputs exactly the same character as `zwnj`. What do you think of using `zwnj` in your layout instead ? You can put the better rendering symbol in `zwnj` instead.

Reply:
Its logical, but I thought the `zwnj` word maybe is a proper noun in a language which maybe is not correct to rename it.

Reply:
Yes it is logical, but I thought maybe the word `zwnj` maybe is a proper noun in a language and maybe its not correct to rename it.

Reply:
`zwnj` is the Unicode name (zero-width non joiner) and is ugly on the keyboard. It's not intended to mean anything in any language. So yes, let's change it for `⸽`.

Reply:
Ok.
https://github.com/Julow/Unexpected-Keyboard/pull/711/commits/cf96ab4b7abe1fdeadcffca36874a6f7dd365d41